### DISCUSSION TITLE
0.8.0 - man, oh podman, edition

### DISCUSSION THREAD

**Discussion by cerebrate** [arkane-systems/bottle-imp#16] [Announcements]
This release includes the following changes:

- Adds remounting of / as shared (this is lost because systemd is running in container mode). Doing so makes podman happy, and probably also other things.

<hr /><em>This discussion was created from the release <a href='https://github.com/arkane-systems/bottle-imp/releases/tag/v0.8.0'>0.8.0 - man, oh podman, edition</a>.</em>

**Comment by mangkoran** [comment]
Thank you as always!

Just one thing I noticed: looks like the pkgver is yet updated for Arch 😆 
```bash
❯ sudo pacman -U bottle-imp-0.8-1-x86_64.pkg.tar.zst
loading packages...
resolving dependencies...
looking for conflicting packages...

Packages (1) bottle-imp-0.7-1

Total Installed Size:  0.03 MiB
Net Upgrade Size:      0.00 MiB

:: Proceed with installation? [Y/n]
```
Other than that, no issue so far. I can confirm that `/` is mounted as shared in this ver.
```bash
❯ findmnt -o PROPAGATION /
PROPAGATION
shared
```

**Reply by mangkoran** [reply]
Nvm https://github.com/arkane-systems/bottle-imp/issues/17

**Comment by cerebrate** [comment]
Yeah, I oopsed and forgot to bump the PKGBUILD. My bad. I'd fix it, but the odds are I'll be doing 0.9 before too long, so... :grin:
